index.js

```diff
@@ -41,7 +41,9 @@ function preset(_, options = {}) {
     opts.include = ['proposal-object-rest-spread'];
 
     // in a web app assume we are using webpack to handle modules
+    // and we want the runtime
     if (target === 'web-app') {
+      opts.runtime = options.runtime == null ? true : options.runtime;
```

Comment:
`defaultOptions.runtime` is `false`, though; is this an issue?

Reply:
no `options` is the passed in value, `opts` the one with defaults applied

Reply:
oh, can we rename this to `explicitOptions` or something then?